$OpenBSD: patch-src_preview_c,v 1.1 2011/11/02 19:12:40 jasper Exp $

Revert "Use new RrButton structure" which was made for new (> ob 3.5) API.
(9cffa6a9ddfc4074f3de8d0302404d70c2818d8f)

--- src/preview.c.orig	Sun Oct 30 16:59:06 2011
+++ src/preview.c	Wed Nov  2 20:03:06 2011
@@ -327,28 +327,28 @@ static GdkPixbuf* preview_window(RrTheme *theme, const
             switch (*layout) {
             case 'D':
                 a = focus ?
-                    theme->btn_desk->a_focused_unpressed :
-                    theme->btn_desk->a_unfocused_unpressed;
+                    theme->a_focused_unpressed_desk :
+                    theme->a_unfocused_unpressed_desk;
                 break;
             case 'S':
                 a = focus ?
-                    theme->btn_shade->a_focused_unpressed :
-                    theme->btn_shade->a_unfocused_unpressed;
+                    theme->a_focused_unpressed_shade :
+                    theme->a_unfocused_unpressed_shade;
                 break;
             case 'I':
                 a = focus ?
-                    theme->btn_iconify->a_focused_unpressed :
-                    theme->btn_iconify->a_unfocused_unpressed;
+                    theme->a_focused_unpressed_iconify :
+                    theme->a_unfocused_unpressed_iconify;
                 break;
             case 'M':
                 a = focus ?
-                    theme->btn_max->a_focused_unpressed :
-                    theme->btn_max->a_unfocused_unpressed;
+                    theme->a_focused_unpressed_max :
+                    theme->a_unfocused_unpressed_max;
                 break;
             case 'C':
                 a = focus ?
-                    theme->btn_close->a_focused_unpressed :
-                    theme->btn_close->a_unfocused_unpressed;
+                    theme->a_focused_unpressed_close :
+                    theme->a_unfocused_unpressed_close;
                 break;
             default:
                 continue;
